src/main/java/http/logger/Logger.java

```diff
@@ -0,0 +1,22 @@
+package http.logger;
```

Comment:
Shouldn't this be in the Utilities package?

Reply:
👍